flycheck.el

```diff
@@ -4694,9 +4738,9 @@ variable symbol for a syntax checker."
           (executable (if current-prefix-arg
                           nil
                         (read-file-name "Executable: " nil default-executable
-                                        nil nil #'executable-find))))
+                                        nil nil #'flycheck-executable-find))))
```

Comment:
Ah, I just noticed that you need to remove `#'` here.  `flycheck-executable-find` is not a function on its own, but just a variable referring to a function.  Hence the function quote is wrong at this place.